### ISSUE TITLE
Export all fails for Android with custom build

### ISSUE THREAD

**Issue by volzhs** [godotengine/godot#42795]
**Godot version:**

3.2.4-beta 6529023

**OS/device including version:**

not related

**Issue description:**

I have 2 export templates for android which uses custom build.
Exporting it one by one works fine.
But Export all fail.
Export path is set to `../release/game_32[64].apk` at export setting.

```
0 param: --xr_mode_regular
1 param: --use_depth_32
2 param: --use_immersive
jarsigner: unable to open jar file: ../release/game_64.apk
ERROR: end_task: Condition "!tasks.has(p_task)" is true.
   At: editor/progress_dialog.cpp:231.
ERROR: _export_all: Failed to export project
   At: editor/project_export.cpp:1025.
0 param: --xr_mode_regular
1 param: --use_depth_32
2 param: --use_immersive
jarsigner: unable to open jar file: ../release/game_32.apk
ERROR: end_task: Condition "!tasks.has(p_task)" is true.
   At: editor/progress_dialog.cpp:231.
ERROR: _export_all: Failed to export project
   At: editor/project_export.cpp:1025.
```

I found that `res://android/release` is created and apks are in there.
This is unexpected location.

**Steps to reproduce:**
1. add 2 android templates
2. enable `custom build` on both templates
3. export all

**Minimal reproduction project:**

**Comment by volzhs** [comment]
cc @m4gr3d

**Comment by akien-mga** [comment]
Is this a regression in 3.2.4 or a pre-existing issue?

**Comment by m4gr3d** [comment]
@akien-mga I think this is a regression caused by the new updates for Android app bundle support. I'm planning to take a look at the issue and resolved them prior to `3.2.4` being finalized.
